modules/outbrainBidAdapter.js

```diff
@@ -291,3 +320,15 @@ function transformSizes(requestSizes) {
 
   return [];
 }
+
+function _getFloor(bidderRequest, type) {
+  const floorInfo = bidderRequest.getFloor({
+    currency: CURRENCY,
+    mediaType: type,
+    size: '*'
```

Comment:
[Prebid documentation](https://docs.prebid.org/dev-docs/modules/floors.html) states that it automatically picks floor for a size if only one size is defined, and uses * if there are multiple.
> To aid in the accuracy of floor selection when using size ”*” in getFloor(), the Price Floors Module has built-in smart rule selection when an ad unit in the internal bidRequest to the bid adapters interface has one ad unit type and one size. In the above example, if the ad unit within the bidRequest object has an ad unit type of “banner” with only one size, say “300x250”, the module will intelligently select the rule with “banner|300x250” in it, as opposed to the “banner|*” rule